db.go

```diff
@@ -74,13 +74,32 @@ var migrations = &migrate.MemoryMigrationSource{
 	},
 }
 
+const (
+	// defaultFwdHistoryLimit is the default limit we place on the forwarding_history table
+	// to prevent creation of an ever-growing table.
+	//
+	// Justification for value:
+	// * ~100 bytes per row in the table.
+	// * Help ourselves to 10MB of disk space
+	// -> 100_000 entries
+	defaultFwdHistoryLimit = 100_000
+)
+
 var defaultNodeKey = route.Vertex{}
 
 type Db struct {
 	db *sql.DB
+
+	fwdHistoryLimit int
+}
+
+func dbWithForwardingHistoryLimit(limit int) func(d *Db) {
```

Comment:
This is actually `dbWithCustomFwdHistoryLimit`, because there's always a limit?

Reply:
I think it's pretty clear that this is a functional option that set the limit you call it with, but happy to rename.